docs/codeql/writing-codeql-queries/introduction-to-ql.rst

```diff
@@ -134,7 +130,9 @@ To import the CodeQL library for a specific programming language, type ``import
    where count(f.getAnArg()) > 7
    select f
 
-➤ `See this in the query console on LGTM.com <https://lgtm.com/query/2096810474/>`__. The ``from`` clause defines a variable ``f`` representing a Python function. The ``where`` part limits the functions ``f`` to those with more than 7 arguments. Finally, the ``select`` clause lists these functions.
+➤ The ``from`` clause defines a variable ``f`` representing a Python function. The ``where`` part limits the functions ``f`` to those with more than 7 arguments. Finally, the ``select`` clause lists these functions.
+
+.. include:: ../reusables/run-query-in-template.rst
```

Comment:
In this case, we probably need to tell them to import and select a Python database before they run the query. (Same for the JavaScript and Java queries below.)

I think we'll need to do this by adding a sentence after the reusable. I did some local testing, and sadly it looks as if we cannot combine the include with other text in a single paragraph. But I think we will need to mention this.

Reply:
Based on direction from @jf205, I removed the reusable and directions to try out the example in the CodeQL template. Is this still valuable information to include after these examples? I could probably test this myself, but right now I'm unsure what is caused by limitations of the template or what is a general improvement to these docs. 💖

Reply:
I think the natural place to tell users to about downloading databases is here: 
https://github.com/github/codeql/blob/siaramist/codeql-template/docs/codeql/writing-codeql-queries/introduction-to-ql.rst?plain=1#L114

You could add a sentence, saying 'For more information about obtaining CodeQL databases, see '[Analyzing your projects](https://codeql.github.com/docs/codeql-for-visual-studio-code/analyzing-your-projects/#choosing-a-database)' in the CodeQL for VS Code documentation.